eng/targets/Wix.Common.props

```diff
@@ -5,9 +5,11 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProductVersion>3.14</ProductVersion>
     <WixVersion>$(MicrosoftSignedWixVersion)</WixVersion>
+    <Wix5Version>$(MicrosoftWixToolsetSdkVersion)</Wix5Version>
```

Comment:
Can you just use the `$(MicrosoftWixToolsetSdkVersion)` property throughout. It should be coming from Arcade at this point.

Reply:
Done